server.js

```diff
@@ -14,10 +17,72 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
 
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+app.get("/messages", (req, res) => {
+  res.json(messages);
+});
+
+app.post("/messages", (req, res) => {
+  if(req.body.from.length == 0){
```

Comment:
Better to use the fact that both empty strings and undefined values are falsey - so you can write `if(!req.body.from)` instead for a nice clean statement

Reply:
Thanks for the feedback, Will look into it at StackOverflow.